modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java

```diff
@@ -107,6 +107,8 @@
 
     protected static final String IMPORT_TMP = "tmpImport";
 
+    protected static final int UPLOAD_SAVE_INTERVAL = 3000;
```

Comment:
Can you add `// save every 3000 documents` to clarify that it's a number of docs and not an interval in seconds or something else